tests/unit/tasks/addon-install-test.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var AddonInstallTask = require('../../../lib/tasks/addon-install');
+var MockProject      = require('../../helpers/mock-project');
+var expect           = require('chai').expect;
+var CoreObject       = require('core-object');
+var Promise          = require('rsvp').Promise;
+
+describe('addon install task', function() {
+  var ui;
+  var project;
+
+  beforeEach(function() {
+    ui = {
+      startProgress: function() {
+
+      }
+    };
+  });
+
+  afterEach(function() {
+    // ui.stopProgress();
+    ui = undefined;
+    project = undefined;
+  });
+
+  describe('when no save flag specified in blueprintOptions', function() {
+    it('calls npm install with --save-dev as a default', function() {
+      var mockNpmInstallTask = CoreObject.extend({
```

Comment:
since this is essentially a class we might want to call this `MockNpmInstallTask` with a captial `M`